ale_linters/go/gometalinter.vim

```diff
@@ -12,6 +13,13 @@ function! ale_linters#go#gometalinter#GetCommand(buffer) abort
     let l:executable = ale_linters#go#gometalinter#GetExecutable(a:buffer)
     let l:filename = expand('#' . a:buffer)
     let l:options = ale#Var(a:buffer, 'go_gometalinter_options')
+    let l:lint_package = ale#Var(a:buffer, 'go_gometalinter_lint_package')
+
+    if l:lint_package
+        return ale#Escape(l:executable)
+        \   . (!empty(l:options) ? ' ' . l:options : '')
+        \   . ' ' . ale#Escape(fnamemodify(l:filename, ':h'))
+    endif
```

Comment:
Maybe do an `if...else` here instead? I know that in Go that is "not done", but this is vimscript so seems like a better pattern in here...

Reply:
This seems largely objective and I find examples within `ale` which use this pattern as it is concise and clean agnostic of the language used. Look at the `scala` linters in `ale`